### PR TITLE
chore: define tailwind theme, add fonts, colors and icons

### PR DESCRIPTION
**Pull request by DerBeton**

### CODE REVIEW

**Comment by Yannic-reust** [comment]
remove dublicate key in tailwind config: line 23 & 25

change line 9 in App.jsx. 

- Why set default class to text-white?
- delete font sonar add, set font in tailwind config -> 

` extend: {
      fontFamily: {
        sans: ['sonar-sans'],
      },
    },`

File: package.json
Comment (Yannic-reust) [inline package.json:1]: Line 17 why is react-helmet-async used?

File: src/App.jsx
Comment (Yannic-reust) [inline src/App.jsx:1]: add font in tailwind config

File: tailwind.config.js
Comment (Yannic-reust) [inline tailwind.config.js:1]: add this to change the font:

`  extend: {
      fontFamily: {
        sans: ['sonar-sans'],
      },
    },`

File: settings.json
Comment (Yannic-reust) [inline settings.json:1]: nice

**Review by Yannic-reust — COMMENTED** [review]
nice work, maybe it can be improved with the commets, otherwise merge confirmed

File: package.json
Comment (DerBeton) [reply package.json:1]: Was used to include font via link-tag in head. Removed in newest commit.

File: src/App.jsx
Comment (DerBeton) [reply src/App.jsx:1]: done

File: tailwind.config.js
Comment (DerBeton) [reply tailwind.config.js:1]: thanks for the tip 🤝

**Pull request merged**